lib/storage_tables/attachable/one.rb

```diff
@@ -14,8 +14,12 @@ class One < ActiveStorage::Attached::One
       #   person.avatar.attach(params[:signed_blob_id]) # Signed reference to blob from direct upload
```

Comment:
Deze supporten we denk ik niet?

Reply:
Nog niet, wil dit wel mogelijk maken.

Reply:
Had je hier al een issue voor aangemaakt?

Reply:
Deze werkt al wel, zie test: https://github.com/payt/storage_tables/blob/1c44251a1474d4ce53560ee86da2919c18763939/test/models/attached/one_test.rb#L91 

De attachment werken nog niet met signed_id's, daarvoor is issue #20